src/lang/english.txt

```diff
@@ -1530,7 +1530,7 @@ STR_CONFIG_SETTING_LINKGRAPH_COLOURS_GREY_TO_RED                :Grey to red
 STR_CONFIG_SETTING_LINKGRAPH_COLOURS_GREYSCALE                  :Greyscale
 
 STR_CONFIG_SETTING_SCROLLMODE                                   :Viewport scroll behaviour: {STRING2}
-STR_CONFIG_SETTING_SCROLLMODE_HELPTEXT                          :Behaviour when scrolling the map
+STR_CONFIG_SETTING_SCROLLMODE_HELPTEXT                          :Behaviour when scrolling the map. The "mouse position locked" options don't work on all systems, like web-based versions, touchscreens, Linux with Wayland, and others
```

Comment:
Do we maybe want to wait until the next run of the web translator so translators at least have a chance to update the string before the release?

Reply:
We need to wait for at least 1 more eints round before we make 13.2, yes. But in general we should do a language backport just before release.
We can force an eints run early, but the question is more: when do we want to release 13.2 :)